kolibri/plugins/learn/assets/src/views/LibraryPage/UnPinnedDevices.vue

```diff
@@ -53,6 +57,11 @@
         required: false,
         default: 0,
       },
+      allDevices: {
+        type: Object,
+        required: false,
+        default: null,
+      },
```

Comment:
With `default: null` here you can run into an issue where if the prop is not provided at all it can result in a `Error: Cannot read properties of null/undefined` in any code that tries to read into `allDevices`, which they assume is an object.

It doesn't look like you're checking for if it's nullish before you try to read from the object's value (ie, your `v-if` around line 7) -- so if `allDevices` really _isn't_ required, then making the default value `{}` _or_ checking if `allDevices` is null before reading from it (or maybe using the `?.`) would be safer.